integration-tests/tests/find-workflow.js

```diff
@@ -46,6 +46,7 @@ const testFindWorkflowFn = _.flow(
     backToTerra()
   ])
 
+  await page.waitForXPath('//*[@id="signInButton"]', { visible: true })
```

Comment:
To confirm page is ready before calling `signIntoTerra()` function.

Encountered pageerror one time while running UI `find-workflow.js` test repeatly on localhost.

<img width="748" alt="Screen Shot 2022-06-27 at 5 32 19 PM" src="https://user-images.githubusercontent.com/35533885/176039093-cd0aef98-b7dd-438c-b161-3847a318f692.png">

Reply:
Why do we need to wait for the sign in button? `signIntoTerra` doesn't use it.

https://github.com/DataBiosphere/terra-ui/blob/261ed6e51c5ec71df3f9120c7f8cf64af2ae0b21/integration-tests/utils/integration-utils.js#L214-L225

Reply:
No, it doesn't. But the new code is not for `signIntoTerra` function. I use it to determine whether page (opened by `backToTerra` function) has finished rendering.

Reply:
It might be clearer to put it in the `backToTerra` function then.